ftdc/custom_format_test.go

```diff
@@ -152,8 +152,13 @@ func TestCustomFormatRoundtripRich(t *testing.T) {
 }
 
 func TestReflection(t *testing.T) {
-	test.That(t, getFieldsForItem(&Basic{100}), test.ShouldResemble,
+	fields, err := getFieldsForItem(&Basic{100})
```

Comment:
`getFieldsForItem` signature changed to include an error. Scary looking no-op diff.